operator/go.mod

```diff
@@ -25,38 +25,50 @@ require (
 	go.opentelemetry.io/otel/trace v1.11.2
 	golang.org/x/exp v0.0.0-20230118134722-a68e582fa157
 	google.golang.org/grpc v1.51.0
-	k8s.io/api v0.25.5
-	k8s.io/apiextensions-apiserver v0.25.5
-	k8s.io/apimachinery v0.25.6
-	k8s.io/apiserver v0.25.5
-	k8s.io/client-go v0.25.5
-	sigs.k8s.io/controller-runtime v0.13.1
+	k8s.io/api v0.26.1
```

Comment:
does this mean we are bumping the libraries to 0.26.1 ? If not I would expect a replace to point to 0.25.x in this file

Reply:
upgrading to 0.26.1 was required to put the adapter into the operator, due to version conflicts with 0.25.x

Reply:
had similar problems here https://github.com/keptn/lifecycle-toolkit/pull/684/files#diff-6141dfe11377eb3de3a7f1903b03e43cad5de00b90b5128f9bf154dfc515972fR30, but I was able to override it with replace https://github.com/keptn/lifecycle-toolkit/pull/684/files#diff-6141dfe11377eb3de3a7f1903b03e43cad5de00b90b5128f9bf154dfc515972fR100 maybe worth trying ?

or are you specifically using some endpoint that is available in 0.26.x API ?